src/components/NavBar.js

```diff
@@ -1,21 +1,22 @@
 import React from 'react';
+import { Link } from 'react-router-dom';
 
 function NavBar() {
   return (
     <footer className="navbar fixed-bottom navbar-light bg-light">
       <nav className="nav nav-fill w-100">
-        <a className="nav-link" href="#month">
+        <Link className="nav-link" to="/month/">
```

Comment:
I like this. I'm not familiar yet with the 'react-router-dom' but I think i get it.